### PR TITLE
Feature/raul valencia html exercise

### PR DESCRIPTION
**Pull request by DkSanjed**
Created exercise of html

### CODE REVIEW

File: 1-the-basics/index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Raul Valencia</title>
+</head>
+<body>
+    <h1 class="header">Raúl Valencia</h1>
+    <h2>Raúl Valencia</h2>
+    <h3>Raúl Valencia</h3>
```
Comment (juancho11gm) [inline 1-the-basics/index.html]: Nice job, you can remove the lines that are repeated. Or you can change the info.

Comment (DkSanjed) [reply 1-the-basics/index.html]: Ok, thanks

**Review by juancho11gm — CHANGES_REQUESTED** [review]
Good job, please see the comments.

**Review by juancho11gm — APPROVED** [review]
Keep it up!

**Pull request merged**